routes/index.js

```diff
@@ -19,49 +19,50 @@ router.get('/inventory', (req, res, next) => {
     res.render('index', { inventory: inventory });
   });
 });
+// get inentory and display all
+
 
 // get login page************************Maryam
 router.get('/login', function(req, res, next) {
   res.render('login');
 });
 
+// update inventory count Manage inventory should move to user.js because of check user sequrity
+// router.put('/inventory/:id', (req, res, next) => {
+//   Inventory.findOne({
+//     _id: req.params.id
+//   }).then(inventory => {
+//     inventory.itemCount = req.body.itemCount;
+//     inventory.save().then(inventory => {
+//       res.json(inventory);
+//     });
+  // });
+// });
+// // post inventory
+// router.post('/inventory', (req, res) => {
+//   const newInventory = {
+//     itemName: req.body.itemName,
+//     itemDepartment: req.body.itemDepartment,
+//     itemPrice: req.body.itemPrice,
+//     itemDescription: req.body.itemDescription,
+//     itemSeller: req.body.itemSeller,
+//     itemCount: req.body.itemCount,
+//     itemImgPath: req.body.itemImgPath
+//   };
+//   new Inventory(newInventory)
+//     .save()
+//     .then(inventory => res.redirect('/inventory'));
+// });
 
```

Comment:
Why did you take all the following code out.   This is the way to add & delete data by using POSTMAN.  It can be also changed to any future use.

Reply:
I just relocated them to user.js file because for inventory edition should check user is able to edit or not and  index.js has not login security but in user.js with /users/inventory method can access all of them after login we can just show inventory items as you said before login